### DISCUSSION TITLE
import rest api via curl

### DISCUSSION THREAD

**Discussion by macosupdate** [Budibase/budibase#1432] [Ideas]
I think you should integrate the feature "import as curl", from chrome. It's already in **appsmith** and **postman**.
It saves a lot of time, when you don't have to type it manually. I have used it a lot
![copy as curl](https://i.imgur.com/YfV5huj.png)

**Comment by joebudi** [comment]
Thanks for this suggestion. I will raise this at the next planning meeting. 

Also, keep the suggestions coming 🙌
